cmd/login/flag.go

```diff
@@ -39,7 +41,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.WCName, flagWCName, "", "Specify the name of a workload cluster to work with. If omitted, a management cluster will be accessed.")
 	cmd.Flags().StringVar(&f.WCOrganization, flagWCOrganization, "", fmt.Sprintf("Organization that owns the workload cluster. Requires --%s.", flagWCName))
 	cmd.Flags().StringSliceVar(&f.WCCertGroups, flagWCCertGroups, nil, fmt.Sprintf("RBAC group name to be encoded into the X.509 field \"O\". Requires --%s.", flagWCName))
-	cmd.Flags().StringVar(&f.WCCertTTL, flagWCCertTTL, "1h", fmt.Sprintf("How long the client certificate should live for. Requires --%s.", flagWCName))
+	cmd.Flags().StringVar(&f.WCCertTTL, flagWCCertTTL, "1h", fmt.Sprintf(`How long the client certificate should live for. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Requires --%s.`, flagWCName))
```

Comment:
Does this helptext print nicely?

Reply:
```
$ go run main.go login --help
Ensure an authenticated context for a Giant Swarm management or workload cluster

Use this command to set up a kubectl context to work with:
  (1) a management cluster, using OIDC authentication
  (2) a workload cluster on AWS or Azure, using client certificate auth

Note that (2) implies (1). When setting up workload cluster access,
management cluster access will be set up as well, if that is not yet done.

Security notes:

Please be aware that the recommended way to authenticate users for
workload clusters is OIDC. Client certificates should only be a
temporary replacement.

When creating client certificates, we recommend to use short
expiration periods (--certificate-ttl) and specific group names
(--certificate-group).

Usage:
  kubectl gs login <management-cluster> [flags]

The <management-cluster> argument can take several forms:

  - the URL of the management cluster Kubernetes endpoint
  - the URL of the Giant Swarm web UI
  - a previously generated context name, with or without the prefix "gs-"

Examples:

Management cluster:

  kubectl gs login https://g8s.test.eu-west-1.aws.gigantic.io

  kubectl gs login gs-mymc

  kubectl gs login mymc

Workload cluster:

  kubectl gs login mymc \
    --workload-cluster gir0y \
    --organization acme \
    --certificate-group admins \
    --certificate-ttl 3h


Flags:
      --as string                      Username to impersonate for the operation
      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
      --cache-dir string               Default HTTP cache directory (default "/home/gerald/.kube/http-cache")
      --callback-port int              TCP port to use by the OIDC callback server. If not specified, a free port will be selected randomly.
      --certificate-authority string   Path to a cert file for the certificate authority
      --certificate-group strings      RBAC group name to be encoded into the X.509 field "O". Requires --workload-cluster.
      --certificate-ttl string         How long the client certificate should live for. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h". Requires --workload-cluster. (default "1h")
      --client-certificate string      Path to a client certificate file for TLS
      --client-key string              Path to a client key file for TLS
      --cluster string                 The name of the kubeconfig cluster to use
      --cluster-admin                  Login with cluster-admin access.
      --context string                 The name of the kubeconfig context to use
  -h, --help                           help for login
      --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
      --internal-api                   Use Internal API in the kube config.
      --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
      --organization string            Organization that owns the workload cluster. Requires --workload-cluster.
      --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
  -s, --server string                  The address and port of the Kubernetes API server
      --tls-server-name string         Server name to use for server certificate validation. If it is not provided, the hostname used to contact the server is used
      --token string                   Bearer token for authentication to the API server
      --user string                    The name of the kubeconfig user to use
      --workload-cluster string        Specify the name of a workload cluster to work with. If omitted, a management cluster will be accessed.

Global Flags:
      --debug                   Toggle debug mode, for seeing full error output.
      --disable-version-check   Disable self-update version check.
```